pvlib/pvsystem.py

```diff
@@ -1373,6 +1379,54 @@ def get_iam(self, aoi, iam_model='physical'):
             raise ValueError(model + ' is not a valid IAM model')
 
 
+class FixedMount:
+    def __init__(self, surface_tilt=0, surface_azimuth=180):
+        self.surface_tilt = surface_tilt
+        self.surface_azimuth = surface_azimuth
+
+    def __repr__(self):
+        return (
+            'FixedMount:'
+            f'\n    surface_tilt: {self.surface_tilt}'
+            f'\n    surface_azimuth: {self.surface_azimuth}'
+        )
+
+    def get_orientation(self, solar_zenith, solar_azimuth):
```

Comment:
Seeing `solar_zenith` and `solar_azimuth` show up here where they're only needed for a common interface makes me think of ABC again.

Maybe `calculate_orientation` or just `orientation`? most of pvlib's `get_` methods/functions wrap a variety of models and have a `model` kwarg.